vitest.setup.ts

```diff
@@ -0,0 +1 @@
+import "@testing-library/jest-dom";
```

Comment:
Needs a different import [for vitest](https://github.com/testing-library/jest-dom?tab=readme-ov-file#with-vitest)
We can also either add the referenced type here for typings or add the types to the tsconfig. I'm fine with either, below is the first option:
```suggestion
/// <reference types="@testing-library/jest-dom" />
import '@testing-library/jest-dom/vitest';
```